### PR TITLE
Display authorized user name

### PR DESCRIPTION
**Pull request by Karakatiza666**
Is this a user-visible change (yes/no): yes

### CODE REVIEW

File: docs/contributors/dev-flow.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ Build and start the Pipeline Manager (that also serves the Feldera Web Console):
 ```bash
 RUST_LOG=info RUST_BACKTRACE=1 cargo run --bin pipeline-manager --features pg-embed -- --api-server-working-directory ~/.dbsp -d postgres-embed --dev-mode --bind-address 0.0.0.0 --sql-compiler-home ./sql-to-dbsp-compiler --dbsp-override-path .  --compiler-working-directory ~/.dbsp --runner-working-directory ~/.dbsp
 ```
-RUST_LOG=info RUST_BACKTRACE=1 AUTH_CLIENT_ID=44jttbufih62p9hh999f3irjkm AUTH_ISSUER="https://cognito-idp.us-east-1.amazonaws.com/us-east-1_gkyurqf2T" AWS_COGNITO_REGION="us-east-1" AWS_COGNITO_USER_POOL_ID="us-east-1_gkyurqf2T" cargo run --bin pipeline-manager --features pg-embed -- --api-server-working-directory ~/.dbsp -d postgres-embed --dev-mode --bind-address 0.0.0.0 --sql-compiler-home ./sql-to-dbsp-compiler --dbsp-override-path .  --compiler-working-directory ~/.dbsp --runner-working-directory ~/.dbsp --auth-provider=aws-cognito
 
 > Here, `~/.dbsp` is the directory that will host compilation artifacts of SQL Compiler that Pipeline Manager will then use. It will be created if it doesn't exist, and you can use another name for it - just replace corresponding arguments in the above command.
```
Comment (lalithsuresh) [inline docs/contributors/dev-flow.md:26]: Not related to this PR, but there's some stale documentation I missed from a previous pass: could you also change `~/.dbsp` to `~/.feldera`?

File: web-console/src/app/(root)/(blank)/auth/aws/page.tsx [hunk 2]
```diff
@@ -19,7 +20,12 @@ export default () => {
     setAuth({
       bearer: params['access_token'],
       user: {
-        username: 'anonymous'
+        avatar: jwtIdPayload['picture'],
```
Comment (lalithsuresh) [inline web-console/src/app/(root)/(blank)/auth/aws/page.tsx:23]: What happens if the JWT payload does not have a picture or these other claims (like phone number...)?

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - SQL: Changed semantics of division to match SQL standard
   ([#1201](https://github.com/feldera/feldera/pull/1201))
+- WebConsole: AWS Cognito username not visible (#1077)
```
Comment (lalithsuresh) [inline CHANGELOG.md:20]: WebConsole: display AWS Cognito username in user profile dropdown (#1077).

File: web-console/src/@core/layouts/components/shared-components/UserDropdown.tsx [hunk 4]
```diff
@@ -65,10 +65,11 @@ const UserDropdown = () => {
         anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
       >
         <Avatar
-          alt='John Doe'
+          alt={'Avatar of ' + auth.user.username}
           onClick={handleDropdownOpen}
           sx={{ width: 40, height: 40 }}
           data-testid='button-avatar-menu'
+          src={auth.user.avatar}
```
Comment (lalithsuresh) [inline web-console/src/@core/layouts/components/shared-components/UserDropdown.tsx:72]: Same question as elsewhere, what happens if they don't have an avatar?

File: web-console/src/app/(root)/(blank)/auth/aws/page.tsx (hunk 2 above)
Comment (Karakatiza666) [reply web-console/src/app/(root)/(blank)/auth/aws/page.tsx:23]: The email / phone are just not shown, and there is just generic icon of a profile image in place of an avatar

**Review by gz — APPROVED** [review]

**Pull request merged**